src/libraries/System.Runtime.InteropServices.RuntimeInformation/tests/DescriptionNameTests.cs

```diff
@@ -30,7 +31,7 @@ public void DumpRuntimeInformationToConsole()
 
             Console.WriteLine($"### FRAMEWORK: Version={Environment.Version} Description={RuntimeInformation.FrameworkDescription.Trim()}");
 
-            string binariesLocation = Path.GetDirectoryName(typeof(object).Assembly.Location);
+            string binariesLocation = PlatformDetection.IsNotBrowser ? Path.GetDirectoryName(typeof(object).Assembly.Location) : "/System.Runtime.InteropServices.RuntimeInformation.Tests.dll";
```

Comment:
do you need this change given that you disable the test?

Reply:
Before this change, it would throw `Drive name must not be empty. (Parameter 'driveName')`, only after the change does it encounter `System.PlatformNotSupportedException : System.Diagnostics.Process is not supported on this platform`. It just seemed more consistent to actually hit PNSE to then disable the test, I can add a comment to `PlatformSpecific` to expand a bit more and remove this change.